deployment/postgres-init-db/sql/expansion/tables/expansion_rules.sql

```diff
@@ -0,0 +1,22 @@
+create table expansion_rules (
+  id integer generated always as identity,
+  activity_type text not null,
+  expansion_logic text not null,
+  expansion_set_id integer not null,
```

Comment:
I think this is a many-to-many association as an expansion function could be associated with any number of expansion sets as command dictionaries and mission models evolve.

Or actually, we might not want the expansion set id here at all as we would be working in the opposite direction of expansion set to expansion rules and want the expansion set to have a list of ids of expansion rules

Reply:
I Schedule a slack huddle to discuss this further with folks.